test/common/deployment_types.go

```diff
@@ -264,22 +112,202 @@ func getDeployment() []Namespace {
 			},
 		},
 	}
-}
-
-func getClusterStorageDeployments() []Namespace {
-	return []Namespace{
+	clusterStorageDeployments = []Namespace{
 		{
 			Name: "redhat-rhmi-operator",
 			Products: []Product{
-				Product{Name: "codeready-postgres-rhmi", ExpectedReplicas: 1},
-				Product{Name: "threescale-backend-redis-rhmi", ExpectedReplicas: 1},
-				Product{Name: "threescale-postgres-rhmi", ExpectedReplicas: 1},
-				Product{Name: "threescale-redis-rhmi", ExpectedReplicas: 1},
-				Product{Name: "ups-postgres-rhmi", ExpectedReplicas: 1},
-				Product{Name: "rhsso-postgres-rhmi", ExpectedReplicas: 1},
-				Product{Name: "rhssouser-postgres-rhmi", ExpectedReplicas: 1},
+				Product{Name: "codeready-postgres-" + InstallationName, ExpectedReplicas: 1},
+				Product{Name: "threescale-backend-redis-" + InstallationName, ExpectedReplicas: 1},
+				Product{Name: "threescale-postgres-" + InstallationName, ExpectedReplicas: 1},
+				Product{Name: "threescale-redis-" + InstallationName, ExpectedReplicas: 1},
+				Product{Name: "ups-postgres-" + InstallationName, ExpectedReplicas: 1},
+				Product{Name: "rhsso-postgres-" + InstallationName, ExpectedReplicas: 1},
+				Product{Name: "rhssouser-postgres-" + InstallationName, ExpectedReplicas: 1},
 				Product{Name: "standard-authservice-postgresql", ExpectedReplicas: 1},
```

Comment:
These looks to be the same as the CRO prefix contants that I externalised for the CRO tests, not sure if you want to use them instead :thinking: 
https://github.com/integr8ly/integreatly-operator/blob/cf24ffb9ea80c1f201e0965beed10e4ee2e6e9a2/pkg/resources/constants/croConstants.go#L4-L11

Reply:
Thanks @KevFan, yes we can use the constants